web/viewer.css

```diff
@@ -1020,23 +1017,23 @@ html[dir='rtl'] .verticalToolbarSeparator {
   margin: 0 0 4px 0;
   height: 1px;
   width: 100%;
-  background-color: hsla(0,0%,0%,.5);
-  box-shadow: 0 0 0 1px hsla(0,0%,100%,.08);
+  background-color: rgba(0, 0, 0, 0.5);
+  box-shadow: 0 0 0 1px rgba(255, 255, 255, 0.08);
 }
 
 .toolbarField {
   padding: 3px 6px;
   margin: 4px 0 4px 0;
   border-radius: 2px;
-  background-color: hsla(0,0%,100%,.09);
-  background-image: linear-gradient(hsla(0,0%,100%,.05), hsla(0,0%,100%,0));
+  background-color: rgba(255, 255, 255, 0.09);
+  background-image: linear-gradient(rgba(255, 255, 255, 0.05), rgba(0, 0, 0, 0));
```

Comment:
The *second* colour value doesn't look correct here; it should be `rgba(255, 255, 255, 0)` as far as I can tell.

Reply:
The converter that I used for these changes (https://bl.ocks.org/njvack/02ad8efcb0d552b0230d) simplified this to `rgba(0, 0, 0, 0)` because the opacity is 0. I chose that converter because it accepts any valid CSS color specification and always outputs RGBA based on the actual value of a rendered pixel. I could obviously change this to `rgba(255, 255, 255, 0)`, but the effect would be the same because of the 0 opacity.

I do agree with you that it's perhaps a bit confusion, so I'll update the patch to use the original values instead. Note that this only applies to values where the opacity component is 0; the other values were already converted as-is, so those have not been changed.

Reply:
Done. This only required changes in the second commit (HSLA to RGBA conversion).

Reply:
> [...] so I'll update the patch to use the original values instead.

Thanks; that did improve readability a tiny bit in the affected places, since it's now immediately obvious that only the alpha channel changes rather than the base colour itself.